sdk/search/Azure.Search.Documents/tests/Samples/HelloWorld.cs

```diff
@@ -4,12 +4,9 @@
 using System;
 using System.Threading.Tasks;
 #region Snippet:Azure_Search_Tests_Samples_Namespaces
-using Azure.Search.Documents;
```

Comment:
Please leave this one in - it's not technically required because our tests are in the same namespace, but we want it showing up in the sample snippet for users.

Reply:
ups. undone